modules/managing-cluster-capabilites.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// *post_installation_configuration/cluster-capabilities.adoc
+
+[id="managing_the_cluster_capabilities_{context}"]
+= Managing the cluster capabilities
```

Comment:
The title is kinda confusing. Seems like managing includes adding, removing, modifying and viewing. And this section looks like `Enabling the cluster capabilities by setting baselineCapabilitySet`.

Reply:
fixed. Updated the title.